lakey_service/downloader/commands.py

```diff
@@ -119,7 +120,8 @@ def post(self, request):
         r.waiters.add(request.access['account'])
 
         if created:
-            r.uri = AthenaExecutor().execute(r)
+            c_item = CatalogueItem.objects.get(id=r.catalogue_item_id)
+            r.uri = c_item.executor.execute(r)
```

Comment:
we can achieve this with more django way: 
```python
r.uri = r.catalogue_item.executor.execute(r)

```
which actually suggests that it may be cleaner to add `execute` at the level of `DownloadRequest` itself making above:

```python
r.execute()
```